src/main/scala/higherkindness/rules_scala/workers/zinc/compile/ZincRunner.scala

```diff
@@ -209,16 +209,14 @@ object ZincRunner extends WorkerMain[ZincRunnerWorkerConfig] {
 
     val compileOptions =
       CompileOptions.create
-        .withSources(sources.map(source => PlainVirtualFile(source.toAbsolutePath().normalize())).toArray)
-        .withClasspath((classesOutputDir +: deps.map(_.classpath)).map(path => PlainVirtualFile(path)).toArray)
+        .withSources(sources.view.map(source => PlainVirtualFile(source.toAbsolutePath().normalize())).toArray)
+        .withClasspath((classesOutputDir +: deps.map(_.classpath)).view.map(path => PlainVirtualFile(path)).toArray)
```

Comment:
Should `deps.map(_.classpath)).view.map(path => PlainVirtualFile(path)).toArray)` instead be `deps.view.map(_.classpath)).map(path => PlainVirtualFile(path)).toArray)`?

Reply:
I tried that, but then realized that I couldn't do it because `classesOutputDir` is prepended to `deps.map(_.classpath)`. Although, now that I think about it, I could do something like:
```scala
(classesOutputDir.view +: deps.view.map(_.classpath)).map(path => PlainVirtualFile(path)).toArray
```

Reply:
Ah, good point. I didn't realize you could prepend to a `View`.